packages/data-stores/src/queries/use-jetpack-search-ai.ts

```diff
@@ -0,0 +1,45 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+
+type AIResponseURL = {
+	url: string;
+	title: string;
+};
+
+interface APIFetchOptions {
+	global: boolean;
+	path: string;
+}
+
+export type JetpackSearchAIResult = {
+	response: string;
+	step: string;
+	urls: AIResponseURL[];
+	terms: string[];
+};
+
+export function useJetpackSearchAIQuery( siteId: number | string, query: string, stopAt: string ) {
+	return useQuery< JetpackSearchAIResult >(
+		[ 'aiQuery', query, stopAt ],
+		() =>
+			canAccessWpcomApis()
+				? wpcomRequest( {
+						path: `sites/${ siteId }/jetpack-search/ai/search`,
+						apiNamespace: 'wpcom/v2/',
+						apiVersion: '2',
+						query: `query=${ encodeURIComponent( query ) }&stop_at=${ stopAt }`,
+				  } )
+				: apiFetch( {
+						global: true,
+						path: `/sites/${ siteId }/jetpack-search/ai/search?query=${ encodeURIComponent(
+							query
+						) }&stop_at=${ stopAt }`,
+				  } as APIFetchOptions ),
```

Comment:
This means we have to test in Atomic, it's a bit of a hassle, but not too bad. Do you need help testing? You'd have to download ETK from Teamcity build of this PR (called Editing Toolkit), upload it as a plugin to an Atomic site after deleting the original, then opening the Help Center.

Reply:
Tried for a bit, but for some reason when testing I just get the spinner after clicking on "Still need help", but nothing happens. No errors that I can see. I'll try again tomorrow.

Reply:
Confirming I tested in atomic (with the feature intentionally not enabled for now). All working as expected.